components/quick_input/quick_input.tsx

```diff
@@ -104,6 +114,24 @@ export class QuickInput extends React.PureComponent<Props> {
                 this.updateInputFromProps();
             }
         }
+        this.onKeyDown();
+    }
+
+    onKeyDown = () => {
```

Comment:
I think we can do this functionality in the `search_bar.tsx` component, where there is already a functionality in place for focusing. That way we leave logic out of this simple component and we avoid scraping the DOM.

Reply:
@nevyangelova search_bar.tsx is the parent only for the input at the top (Globar Header search bar) and not for textBox at the bottom (post message textBox), so it does not have any way to know when the focus is out of the postMessage textbox so that we can set the next focus to the Globar Header search bar when F6 is pressed.
can u suggest something here please?

Reply:
@AbhinavVihan I just checked that this bug only shows up in the windows Desktop app version. I don't have a Win machine but the focus works correctly, except using JAWS. So I would suggest to not add additional logic for focusing, especially not manually removing focus from one input to another. F6 is the focusing key for electron apps by default, so what its missing is probably an issue with ordering or a missing prop that JAWS requires. Some research into windows and JAWS might be necessary here

Reply:
sorry, but i don't have a windows system either.

Reply:
@AbhinavVihan would you like to grab another accessibility issue in this case? We can specify that we need a windows machine for this one. Sorry about that!

Reply:
Yeah Sure.
Thank you